File: daraja/src/main/java/com/androidstudy/daraja/network/LNMAPI.java
package com.androidstudy.daraja.network;

import com.androidstudy.daraja.model.LNMExpress;
import com.androidstudy.daraja.model.PaymentResult;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface LNMAPI {

    @POST("mpesa/stkpush/v1/processrequest")
    Call<PaymentResult> getLNMPesa(@Body LNMExpress lnmExpress);

}

File: daraja/src/main/java/com/androidstudy/daraja/network/LNMAPI.kt
package com.androidstudy.daraja.network

import com.androidstudy.daraja.model.LNMExpress
import com.androidstudy.daraja.model.PaymentResult
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.POST

interface LNMAPI {

    @POST("mpesa/stkpush/v1/processrequest")
    fun getLNMPesa(@Body lnmExpress: LNMExpress): Call<PaymentResult>
}